infra/modules/azure_function_app/main.tf

```
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.103.1"
```

```suggestion
      version = "~> 3.103.1"
```
Do you think it may be beneficiary to employ the same kind of constraint used in azurermv3?

---

Accorded